tests/elk/logstash/statefulset.yaml

```diff
@@ -30,15 +30,15 @@ spec:
         fsGroup: 1000
         runAsUser: 1000
       affinity:
-        podAntiAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            - labelSelector:
-                matchExpressions:
-                  - key: app
-                    operator: In
-                    values:
-                      - "logstash"
-              topologyKey: kubernetes.io/hostname
+#        podAntiAffinity:
```

Comment:
if re we moved logstash shouldn't we remove this whole folder?

Reply:
leaving it for future utilisation, in case we will be needing it.
it's in a separate folder so it doesn't affect the code complexity.

Reply:
if you'd like I can delete this